workflow-runtime-android/src/androidTest/java/com/squareup/workflow1/android/AndroidDispatchersRenderWorkflowInTest.kt

```diff
@@ -0,0 +1,415 @@
+@file:OptIn(WorkflowExperimentalRuntime::class)
+@file:Suppress("JUnitMalformedDeclaration")
+
+package com.squareup.workflow1.android
+
+import app.cash.burst.Burst
+import com.squareup.workflow1.RuntimeConfig
+import com.squareup.workflow1.RuntimeConfigOptions.CONFLATE_STALE_RENDERINGS
+import com.squareup.workflow1.RuntimeConfigOptions.Companion.RENDER_PER_ACTION
+import com.squareup.workflow1.RuntimeConfigOptions.PARTIAL_TREE_RENDERING
+import com.squareup.workflow1.RuntimeConfigOptions.RENDER_ONLY_WHEN_STATE_CHANGES
+import com.squareup.workflow1.RuntimeConfigOptions.STABLE_EVENT_HANDLERS
+import com.squareup.workflow1.Workflow
+import com.squareup.workflow1.WorkflowExperimentalRuntime
+import com.squareup.workflow1.WorkflowInterceptor
+import com.squareup.workflow1.WorkflowInterceptor.RenderPassesComplete
+import com.squareup.workflow1.WorkflowInterceptor.RuntimeLoopOutcome
+import com.squareup.workflow1.action
+import com.squareup.workflow1.asWorker
+import com.squareup.workflow1.renderChild
+import com.squareup.workflow1.runningWorker
+import com.squareup.workflow1.stateful
+import com.squareup.workflow1.ui.renderWorkflowIn
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.drop
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.plus
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import papa.Choreographers
+import java.util.concurrent.atomic.AtomicInteger
+import kotlin.test.assertEquals
+
+@OptIn(WorkflowExperimentalRuntime::class)
+@Burst
+class AndroidDispatchersRenderWorkflowInTest(
+  private val runtime: RuntimeOptions = RuntimeOptions.DEFAULT
```

Comment:
Note that burst will create a separate test class for each runtime config combination and run them separately. (i'll need to migrate my hand rolled parameterization in the other tests for runtime config to this soon).